README.md

```diff
@@ -63,10 +187,33 @@ You can also right-click on any element in the ARM Template Outline view to init
   - [resourceId() function](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#resourceid)
```

Comment:
I'd say built-in functions and give resourceId/etc as examples.

Maybe also say resourceId arguments separately?

Reply:
Fixed